actions/class.RestQtiTests.php

```diff
@@ -252,6 +252,6 @@ private function getTestClass()
      */
     private function getQtiPackageExporter()
```

Comment:
Is it possible to add return type hints?

Reply:
Added